src/playback-watcher.js

```diff
@@ -79,6 +79,12 @@ export default class PlaybackWatcher {
    * @private
    */
   checkCurrentTime_() {
+    if (this.tech_.seeking() && this.fixBadSeeks_()) {
```

Comment:
Can we please, please rename `fixBadSeeks_` to `fixesBadSeeks_` because that would make me soooo happy.